Lib/test/test_zipfile.py

```diff
@@ -2496,11 +2496,15 @@ def test_read_zip64_with_exe_prepended(self):
         self._test_zip_works(self.exe_zip64)
 
     @unittest.skipUnless(sys.executable, 'sys.executable required.')
+    @unittest.skipUnless(os.access('/bin/bash', os.X_OK),
```

Comment:
How about using shutil.which('bash') instead? There are some windows machines with bash installed now... will the test work there?

Reply:
I didn't think windows supported #! line executables as a concept.  executable is based on filename ending in bat/cmd/com/exe, not an execute bit there.